### ISSUE TITLE
Arkansas Data 2/28

### ISSUE THREAD

**Issue by CSSEGISandData** [CSSEGISandData/COVID-19#3738]
Hello all,

This serves as an announcement that here will be abnormal data for the US state of Arkansas in the update now being propagated to the dashboard and being published in our data products later today. The Arkansas Department of Health published an announcement on their [twitter page here](https://twitter.com/ADHPIO/status/1366163333225799682).

This is the text:

"A data clean-up is reflected in today's reports. Some deaths have been reclassified and duplicates were removed. 288 new cases are added today. The remaining 2,932 additional cases occurred prior to February 14, 2021."

In summary, cases are spiking due to 2,932 backlogged cases and deaths are falling by 174 due to duplicated reports. These anomalies are occurring across numerous counties.

**Comment by CSSEGISandData** [comment]
Hello all,

Unfortunately Arkansas does not have internal data that we will be able to use to correct our dataset.
